assets/prototype/domain/eventEditor/containers/useConfigProvider.ts

```diff
@@ -0,0 +1,30 @@
+import { useContext } from 'react';
+import { pick } from 'ramda';
+import { ConfigContext } from '../../../infrastructure/services/contextProviders/ConfigProvider';
+
+type ConfigsToBeRetrieved = {
```

Comment:
CNO @tn3rb I might need a better naming here, open to suggestions.

Reply:
I think we should standardize the naming for all type declarations and interfaces.

Since we are type casting (not sure if that is the correct term when using TS?) the props for a component/hook, then the naming schema should be:

```js
`${componentOrHookName}Props`
```

So in this case, the prop types for `useConfigProvider` should simply be named `useConfigProviderProps`

This way, no one will ever have to guess or think about what the corresponding props type is.